_guidelines-v4/01-introduction/01-about/01-designprinciples.md

```diff
@@ -0,0 +1,41 @@
+---
+version: "v4"
+title: "MEI Design Principles"
+sectionid: "designprinciples"
+---
+
+This section of the Guidelines defines principles and criteria for designing, developing, and maintaining an XML-based encoding scheme for music notation documents.
+
+##### Definitions and Parameters
+
+A music notation document is one that contains music notation; that is, any one of a number of "visual analogues of musical sound, either as a record of sound heard or imagined, or as a set of visual instructions for performers." (Ian D. Bent, et al. "Notation." Grove Music Online. Oxford Music Online. 25 May 2010. [http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114](http://www.oxfordmusiconline.com/subscriber/article/grove/music/20114){:target='\_blank'}.)
```

Comment:
There are also haptic or other-modal equivalents than just "visual" (cf. Braille). Bents "definition" is quite narrow in this sense.

Reply:
Thanks for the addition